packages/tc-api-rest-handlers/lib/relationships/write.js

```diff
@@ -3,6 +3,7 @@ const { stripIndents } = require('common-tags');
 const { getType } = require('@financial-times/tc-schema-sdk');
 const { metaPropertiesForCreate } = require('../metadata-helpers');
 const { findInversePropertyNames } = require('./properties');
+const { metaPropertiesForUpdate } = require('../metadata-helpers');
```

Comment:
`metaPropertiesForCreate` and `metaPropertiesForUpdate` can combine with same time to require

Reply:
😮 oh, thank you for letting me know! I will combine them.